src/v2/Components/Artwork/Details.tsx

```diff
@@ -230,14 +239,22 @@ export const Details: React.FC<DetailsProps> = ({
             )}
         </Flex>
       )}
-      {!hideArtistName && <ArtistLine {...rest} />}
+      <Flex flexDirection="row" justifyContent="space-between">
+        {!hideArtistName && <ArtistLine {...rest} />}
+        {shouldShowHoverSaveButton && (
+          <NewSaveButtonFragmentContainer
+            contextModule={ContextModule.artworkGrid}
+            artwork={rest.artwork}
+          />
+        )}
```

Comment:
In `<GridItem />` we never pass the `hideArtistName` (see [here](https://github.com/artsy/force/blob/f022c3d8d0c26aa345b848e7f036e846137e2305/src/v2/Components/Artwork/GridItem.tsx#L112)). So this is why I didn't consider that.

Am I missing any case were this would cause a problem?

Reply:
> In <GridItem /> we never pass the hideArtistName (see [here](https://github.com/artsy/force/blob/f022c3d8d0c26aa345b848e7f036e846137e2305/src/v2/Components/Artwork/GridItem.tsx#L112)). So this is why I didn't consider that.

 ahh. good point! I forgot about the fact that the changes are only available in `GridItem` component

Reply:
Thanks for checking